lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+The Libra Payment Network (“LPN”) is a blockchain-backed payment platform and an infrastructure for stablecoins. Both the blockchain and the stablecoin financial instruments are controlled by Libra Networks (“Libra Networks” or “LN”) a subsidiary of the Libra Association (the “Association” or “LA”). Libra Networks is in the process of applying for a payment system license from the Swiss Financial Market Supervisory Authority (“FINMA”). Once granted, LN will be responsible for ensuring that LPN operates in compliance with the payment system license from FINMA (the "FINMA License").
+
+The Libra Association is an independent membership organization that is responsible for the governance of the LPN and the development of the Libra project. The Association is governed by the Association Council, which is comprised of one representative per Association member. The Association Council manages the technology roadmap of the platform and oversees and maintains the Libra Improvement Proposal and Libra Upgrade Process, often through subcommittees of the Council such as the Technical Steering Committee (the “TSC”) and the Association Board.
+
+Each member of the Libra Association is expected to be ready to run a replica of the Libra protocol, called a Validator Node. Validator Nodes coordinate and reach consensus on the state of the blockchain by processing transactions, they ensure correct updates even if some Validator Nodes (fewer than a third) fail in arbitrary ways. Though only a subset of members, called a Validator Set, run Validator Nodes in any given time. The Association determines the composition of the Validator Set by adding and removing members over time. Reasons for changing the Validator Set include new members joining the Association or existing members leaving it and actions needed to comply with LN’s obligations under its FINMA License designed to guarantee the secure and performant operation of the platform. The Validator Set is stored as part of the blockchain state, in an on-chain configuration. 
+
+# Terminology
+* *Validator Owner* is an owner of the Libra account of the Libra Association's member. The account stores the *Validator Config* resource.
+* *Validator Config* is an on-chain resource that contains information for running a Validator Node. It includes a *consensus public key*, which is an EdDSA public key used to authenticate consensus messages. It also includes *network information*: network addresses and static public keys for establishing secure connections with this Validator Node from other Validator Nodes or from clients.
+* *Validator Set* is an on-chain resource that stores a vector, where each entry contains an address of a Validator Owner and a copy of its Validator Config. Though the Validator Config in the Validator Set may be an older version of the Validator Config currently stored under Validator Owner's account, each Validator Node is run with the Validator Config from the Validator Set.
```

Comment:
Validator set defines the current set of validators performing the consensus protocol. Specifically it is a vector containing validator config entries that map to the validators current states. These entries may differ...